furms-web-ui/src/main/java/io/imunity/furms/ui/user_context/RoleTranslatorService.java

```diff
@@ -108,4 +111,8 @@ public RoleTranslatorService(SiteService siteService, CommunityService community
 				throw new IllegalArgumentException("This shouldn't happen, viewMode level should be always declared");
 		}
 	}
+
+	private String getName(String name) {
+		return name + " admin";
```

Comment:
@bihu1  I'm merging this to dev, but please add on the toto list that the "admin" suffix should be localized.